packages/ra-core/src/core/useGetResource.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import { ResourceContextValue } from './ResourceContext';
+import { useResources } from './useResources';
+
+export const useGetResource = (name: string): ResourceContextValue => {
```

Comment:
without jsDoc, this one is confusing. Why would you want to use it instead of useResourceConfig?